mlrun/run.py

```diff
@@ -585,7 +585,7 @@ def new_function(
                      (job, mpijob, ..) the handler can also be specified in the `.run()` command, when not specified
                      the entire file will be executed (as main).
                      for nuclio functions the handler is in the form of module:function, defaults to "main:handler"
-    :param source:   valid path to git, zip, or tar file, e.g. `git://github.com/mlrun/something.git`,
+    :param source:   valid absolute path to git, zip, or tar file, e.g. `git://github.com/mlrun/something.git`,
                      `http://some/url/file.zip`
```

Comment:
I am not sure about the order of the comment, `absolute path to git` sounds a bit strange to me, wdyt?